mintpy/smallbaselineApp.py

```diff
@@ -1063,8 +1065,72 @@ def plot_result(self, print_aux=True, plot=True):
+            tropo_model = self.template['mintpy.troposphericDelay.weatherModel']
+            geo_dir = os.path.join(self.workDir, 'geo')
+
+            # view options
+            opt1 = ['--nodisplay', '--dpi', '150', '--update']
+            opt2 = ['--nodisplay', '--dpi', '150', '--update', '--mask', mask_file, '--noaxis', '-u', 'cm', '--wrap-range', '-10', '10']
+
+            iargs_list = [
+                ['velocity.h5'] + opt1,
+                ['temporalCoherence.h5'] + opt1 + ['-c', 'gray', '--vlim', '0', '1'],
+                ['maskTempCoh.h5'] + opt1 + ['-c', 'gray', '--vlim', '0', '1'],
+                [dem_file] + opt1,
+                [ifgram_stack] + opt1 + ['-unwrapPhase', '--zero-mask', '--wrap'],
+                [ifgram_stack] + opt1 + ['-unwrapPhase', '--zero-mask'],
+                # [ifgram_stack] + opt1 + ['-coherence', '--mask', 'no'],
+                # [ifgram_stack] + opt1 + ['-connectComponent', '--mask', 'no'],
+                [ifgram_stack] + opt1 + ['-unwrapPhase_bridging', '--zero-mask'],
+                [ifgram_stack] + opt1 + ['-unwrapPhase_closure', '--zero-mask'],
+                [ifgram_stack] + opt1 + ['-unwrapPhase_bridging_phaseClosure', '--zero-mask'],
+                ['avgPhaseVelocity.h5'] + opt1,
+                ['avgSpatialCoh.h5'] + opt1 + ['-c', 'gray', '--vlim', '0', '1'],
+                ['maskConnComp.h5'] + opt1 + ['-c', 'gray', '--vlim', '0', '1'],
+                ['timeseries.h5'] + opt2,
+                ['timeseries_LODcor.h5'] + opt2,
+                ['timeseries_LODcor_ECMWF.h5'] + opt2,
+                ['timeseries_LODcor_ECMWF_demErr.h5'] + opt2,
+                ['timeseries_LODcor_ECMWF_ramp.h5'] + opt2,
+                ['timeseries_LODcor_ECMWF_ramp_demErr.h5'] + opt2,
+                [f'timeseries_{tropo_model}.h5'] + opt2,
+                [f'timeseries_{tropo_model}_demErr.h5'] + opt2,
+                [f'timeseries_{tropo_model}_ramp.h5'] + opt2,
+                [f'timeseries_{tropo_model}_ramp_demErr.h5'] + opt2,
+                [f'timeseries_ramp.h5'] + opt2,
+                [f'timeseries_demErr_ramp.h5'] + opt2,
+                [f'timeseries_demErr.h5'] + opt2,
+                [os.path.join(geo_dir, 'geo_maskTempCoh.h5')] + opt1 + ['-c', 'gray'],
+                [os.path.join(geo_dir, 'geo_temporalCoherence.h5')] + opt1 + ['-c', 'gray'],
+                [os.path.join(geo_dir, 'geo_velocity.h5')] + opt1 + ['velocity'],
+                [os.path.join(geo_dir, 'geo_timeseries_ECMWF_demErr_ramp.h5')] + opt1 + ['--noaxis'],
+                [os.path.join(geo_dir, 'geo_timeseries_ECMWF_demErr.h5')] + opt1 + ['--noaxis'],
+                [os.path.join(geo_dir, 'geo_timeseries_demErr_ramp.h5')] + opt1 + ['--noaxis'],
+                [os.path.join(geo_dir, 'geo_timeseries_demErr.h5')] + opt1 + ['--noaxis'],
+                [f'velocity{tropo_model}.h5'] + opt1 + ['--mask', 'no'],
+                [f'numInvIfgram.h5'] + opt1 + ['--mask', 'no']
```

Comment:
May I ask why do you add `f` to some files?

Reply:
I used f-strings for string formatting. In your README it says MintPy works with Python 3.6+. F-strings were introduced in Python 3.6. Do you prefer I use a specific style for string formatting?

I used it like this below:
```python
# Loaded model is 'ERA5'
tropo_model = self.template['mintpy.troposphericDelay.weatherModel']

# String becomes velocityERA5.h5
f'velocity{tropo_model}.h5'
```

Reply:
I see, this's great.